examples/buy-lbtc-with-usdt/main.go

```diff
@@ -0,0 +1,313 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"log"
+	"math"
+	"time"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/btcec/v2/ecdsa"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/google/uuid"
+	tdexv2 "github.com/tdex-network/tdex-daemon/api-spec/protobuf/gen/tdex/v2"
+	"github.com/tdex-network/tdex-daemon/pkg/explorer"
+	"github.com/tdex-network/tdex-daemon/pkg/explorer/esplora"
+	"github.com/vulpemventures/go-elements/elementsutil"
+	"github.com/vulpemventures/go-elements/network"
+	"github.com/vulpemventures/go-elements/payment"
+	"github.com/vulpemventures/go-elements/psetv2"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	client      tdexv2.TradeServiceClient
+	explorerSvc explorer.Service
+
+	lbtc         = network.Regtest.AssetID
+	traderAmount = 10.0
+)
+
+func initEnv() error {
+	conn, err := grpc.Dial("localhost:9945", grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		return fmt.Errorf("failed to connect to daemon: %s", err)
+	}
+	client = tdexv2.NewTradeServiceClient(conn)
+
+	explorerSvc, err = esplora.NewService("http://localhost:3001", 15000)
+	if err != nil {
+		return fmt.Errorf("failed to prepare explorer: %s", err)
+	}
+	return nil
+}
+
```

Comment:
Should we add doc that in order to run example it is needed to start daemon and setup market? or maybe make cmd(invoking script) that starts daemon with neccessery data?

Reply:
My idea is to use these examples in the e2e test, which is yet to be fixed. Just wanted to do it in a separate PR.

Also, I didn't add a README for them because we would need to describe all the setup to open a market etc., while we already have the [dev-portal](https://dev.tdex.network/docs/provider/) for this, which, is yet to be updated too.